vcpkg/TOC.yml

```diff
@@ -30,6 +30,14 @@
       href: users/authentication.md
     - name: Authenticate to private Git registries in your GitHub Actions workflows
       href: consume/gha-authentication.md
+  - name: Reduce build times with binary caching
+    items:
+    - name: "What is binary caching?"
+      href: consume/binary-caching-overview.md

```

Comment:
This sounds like it should be a concept article.

Reply:
This one's following the "Overview" template